crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/FindCommand.java

```diff
@@ -121,13 +121,18 @@ public void write(OutputStream os) throws IOException {
                     writer.flush();
                 } finally {
                     streamResponse.documentStream.close();
+                    if (context != null) {  
+                        context.endRequestMonitoring();
+                    }
+                    // TODO: What if there is IOException?
```

Comment:
Still need to mark any errors that occur here I think. Otherwise I suspect they'll bubble up to RESTEasy and get lost.